### PR TITLE
Update device provisioning service documentation

### PR DESCRIPTION
**Pull request by Danielius1922**
## Summary by CodeRabbit

- **Documentation**
  - Removed outdated references to the Device Provisioning Service (DPS) in deployment and troubleshooting documentation, streamlining focus towards the Hub service.
  - Updated deployment commands and configuration references to reflect changes in service structure and access methods for Docker images.
  - Simplified deployment instructions by consolidating focus on the `plgd-hub` while ensuring the device provisioning service is still configured to be enabled.
  - Enhanced clarity and usability of the Device Provisioning Service (DPS) documentation within the IoTivity-lite framework, including refined initialization processes and improved error handling.
  - Updated references to header files for improved clarity and accessibility.

### CODE REVIEW

File: content/en/docs/configuration/device-provisioning-service.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ CoAP API as specified in the [workflow](./workflow.puml).
 
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
-| `apis.coap.address` | string | `Listen specification <host>:<port> for coap client connection.` | `"0.0.0.0:5688"` |
+| `apis.coap.address` | string | `Listening specification <host>:<port> for coap client connection.` | `"0.0.0.0:5688"` |
```
Comment (jkralik) [inline content/en/docs/configuration/device-provisioning-service.md:41]: 0.0.0.0:15684 ?

**Review by jkralik — APPROVED** [review]

File: content/en/docs/configuration/device-provisioning-service.md (hunk 1 above)
Comment (Danielius1922) [reply content/en/docs/configuration/device-provisioning-service.md]: Updated

File: content/en/docs/configuration/device-provisioning-service.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ The Device Provisioning Service provides API to provision device to the [plgd/hu
 Before you use the image you need to setup [K8s access to private registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry).
 
 ```bash
-docker pull ghcr.io/plgd-dev/device-provisioning-service/device-provisioning-service:latest
+docker pull ghcr.io/plgd-dev/hub/device-provisioning-service:latest
```
Comment (Danielius1922) [inline content/en/docs/configuration/device-provisioning-service.md:17]: This doesn't work right know, since we haven't made a realease yet with DPS.

Comment (jkralik) [reply content/en/docs/configuration/device-provisioning-service.md:17]: We can release it now :)

**Comment by jkralik** [comment]
Remove all `helm` commands with `plgd-dps` and also maybe `kubectl` with `dps-plgd`. Also in kubectl commands `hub-plgd` need to be changed to `plgd-hub`.

File: content/en/docs/deployment/device-provisioning-service/troubleshooting.md [hunk 3]
```diff
@@ -27,14 +27,12 @@ If you encounter issues with the Device Provisioning Service or Hub, follow thes
 
    ```sh
    helm upgrade -i -n plgd --create-namespace -f ./withUpdatedMock.yaml hub plgd/plgd-hub
-   helm upgrade -i -n plgd --create-namespace -f ./withUpdatedMock.yaml dps plgd/plgd-dps
    ```
 
 3. Restart the pods by deleting them:
 
    ```sh
    kubectl -n plgd delete $(kubectl -n plgd get pods -o name | grep "hub-plgd")
```
Comment (jkralik) [inline content/en/docs/deployment/device-provisioning-service/troubleshooting.md:35]: replace `grep "hub-plgd"` -> `grep "plgd-hub"`

Comment (Danielius1922) [reply content/en/docs/deployment/device-provisioning-service/troubleshooting.md]: fixed

**Pull request merged**